src/Orckestra.Media.AutoImageResizing/ImageResizer.cs

```diff
@@ -44,7 +43,9 @@ private static void ConvertImageToPictureTag(XElement imgElement)
 
             if (AutoImageResizingHelper.IsLocalC1MediaWithoutResizingOptions(imageSrc))
             {
-                var pictureElement = new XElement("picture");
+                var imageNamespace = imgElement.GetDefaultNamespace();
+                var pictureElement = new XElement(imageNamespace + "picture");
```

Comment:
Instead of the default namespace, it should be "http://www.w3.org/1999/xhtml".

```
        private static readonly XName PictureTagXName = Namespaces.Xhtml + "picture";
        private static readonly XName SourceTagXName = Namespaces.Xhtml + "source";
        private static readonly XName ImgTagXName = Namespaces.Xhtml + "img";
```

```
var pictureElement = new XElement(PictureTagXName);
```